CHANGELOG.md

```diff
@@ -1,3 +1,308 @@
+<a name="2.0.0-beta.9"></a>
+# [2.0.0-beta.9 découpage-panjandrum](https://github.com/angular/material2/compare/2.0.0-beta.8...2.0.0-beta.9) (2017-08-29)
+
+### Highlights
+* Over 140 bug fixes
+* Nested menus
+* Autocomplete supports `md-optgroup`
+* `Overlay` moved to `@angular/cdk`
+* New `MdSelectionList`
+* Almost all components now use `OnPush` change detection (dialog being the exception).
+* You can now get back the `EmbeddedViewRef` when attaching a `TemplatePortal`
+* `MdSidenav` has been split into `MdSidenav` and `MdDrawer`. The `MdSidenav` is now meant to be
+used for top-level application navigation, while the drawer is meant to be used for more local
+split views. While there are no differences introduced between the two in this release, future
+releases will see different features added to each.
+
+### Breaking changes
+* Imports from `@angular/cdk` are now scoped to a specific sub-package. For example, if you
+previous had:
+```ts
+import {LiveAnnouncer, Overlay, Directionality} from '@angular/cdk';
+```
+You would will now need to write:
+```ts
+import {LiveAnnouncer} from '@angular/cdk/a11y';
+import {Directionality} from '@angular/cdk/bidi';
+import {Overlay} from '@angular/cdk/overlay';
+```
+
+This helps ensure you're only pulling in the pieces of the cdk being used as well as providing more
+context about what an imported symbol is be used for.
+
+The current set of public `@angular/cdk` subpackages are:
+a11y, bidi, coercion, collections, keycodes, observers, overlay, platform, portal, rxjs,
+scrolling, table.
+
+If you are using SystemJS, each package you use must be added to the SystemJS configuration. 
+* All `Overlay` code has been moved from `@angular/material` to `@angular/cdk`. The symbols are
+still re-exported through `@angular/material`, but these re-exports will be removed in a
+subsequent release.
+* `cdkScrollable`, `ScrollDispatcher`, and `ViewportRuler` have been moved from overlay into its
+own `scrolling` subpackage in `@angular/cdk`. 
+* **input:** `md-prefix` and `md-suffix` are now `mdPrefix` and `mdSuffix`.
+* **portal:** `TemplatePortal` now requires a generic type (C) to align with `TemplateRef`.
+This will usually be `any`.
+* **portal:** Attaching a template portal will now return an `EmbeddedViewRef<C>` instead of an
+empty `Map`.
+* **observe-content:** `ObserveContentModule` has been renamed to `ObserversModule`
+* **overlay:** `PositionStrategy` now has an `attach` method and no longer passes the overlay
+DOM element to `apply`.
+* **datepicker:** You must now use a date object (of whatever type your DateAdapter uses) rather
+than a string when setting the value of the datepicker programmatically (through value, ngModel,
+or formControl).
+* **datepicker:** `NativeDateAdapter` will now use Angular's LOCALE_ID instead of the browser's
+locale.
+* **sidenav:** CSS classes have changed from `mat-sidenav-` to `mat-drawer-`
```

Comment:
Nit: CSS class prefixes ?

Reply:
I consider the "prefix" to just be "md"

Reply:
Don't forget about `mdDatepickerToggle` (looking for the thread now)